dome9/resource_dome9_continuous_compliance_notification.go

```diff
@@ -700,3 +858,17 @@ func flattenEmailPerFindingData(respEmailPerFindingData *continuous_compliance_n
 
 	return []interface{}{m}
 }
+
+func ValidatePayloadFormatJSON(v interface{}, k string) (ws []string, errors []error) {
```

Comment:
is this ok for empty json on null values?

Reply:
yep